.travis.yml

```diff
@@ -37,7 +112,10 @@ script:
       mkdir _build                        &&
       cd _build                           &&
       { ../configure $CONFIGURE_FLAGS || { cat config.log; exit 1; } ; } &&
-      make -j2                            &&
-      make -j2 check                      &&
-      make -j2 distcheck DISTCHECK_CONFIGURE_FLAGS="$CONFIGURE_FLAGS";
+      if [ -z "$DISTCHECK_ONLY" ]; then
+        make -j2 VERBOSE=1 -n                           &&
+        make -j2 check;
+      else
+        make -j2 distcheck DISTCHECK_CONFIGURE_FLAGS="$CONFIGURE_FLAGS";
+      fi;
```

Comment:
I don't like that, because what you do is build one more time, and only that one do the distcheck.  And given how distcheck works (make the tarball and try to build it), it could possibly be affected by the configuration options selecting buildig or not some code (even though it should not if we didn't mess up, but CI is to catch case where we did, isn't it :smile: ).  So it is a good thing it's checked for all variations of the environment.

Only the same environment with different compilers is not useful, because building with another compiler shouldn't affect *what* we're building.

Reply:
I didn't completely understand this but let me know if the change I made is what you want.

Reply:
> Now its failed on another `noexcept`, @andy5995 please don't trigger another build, its not possible to see old build logs, wait for others to see it.

Sure, I'll wait. But I do believe you can see old build logs at https://travis-ci.org/geany/geany/pull_requests

Reply:
> I didn't completely understand this but let me know if the change I made is what you want.

now `DISTCHECK_ONLY` is never set, is it?
but stop worrying about this part and we'll see once the build stuff is dealt with

Reply:
my bad I missed the one in the matrix